lib/feature_flagger/storage/redis.rb

```diff
@@ -50,6 +71,32 @@ def all_values(key)
       def search_keys(query)
         @redis.scan_each(match: query)
       end
+
+      private
+
+      def resource_key(resource_name, resource_id)
+        FeatureFlagger::Storage::RedisKeys.resource_key(
+          RESOURCE_PREFIX,
+          resource_name,
+          resource_id,
+        )
+      end
+
+      def remove_feature_key_from_resources(feature_key)
+        cursor = 0
+
+        loop do
+          cursor, resource_keys = @redis.scan(cursor, match: "#{RESOURCE_PREFIX}:*", count: SCAN_EACH_BATCH_SIZE)
+          
+          @redis.multi do |redis|
+            resource_keys.each do |key|
+              redis.srem(key, feature_key)
```

Comment:
SREM accepts an array of keys. Even though it will increase the time complexity, it'll save several network roundtrips.

Reply:
👍

Reply:
actually the variant argument here is the key, not the feature_key.